### ISSUE TITLE
Revisit Logger constraints in Cats-Effect 3

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#4131]
In eca20a5db4b, we had a clash between `Sync` and `Concurrent` constraints.  I ignorantly promoted them all to `Async` to make things compile.  Maybe there was a better way.

**Comment by rossabaker** [comment]
I think this is related to @ChristopherDavenport's concern about no longer being able to run Natchez with Ember.

**Comment by rossabaker** [comment]
This was a giant merge.  I don't even know what module I was talking about.  Probably Ember, because log4s doesn't affect constraints?

**Comment by m-sp** [comment]
yes this was about ember but not sure if this is really fixable, would be good to get @ChristopherDavenport feedback here

**Comment by bplommer** [comment]
Is this still relevant? I think going up to `Async` is the Blessed way to get both `Sync` and `Concurrent`.

**Comment by armanbilge** [comment]
Arguably, yes. I'd argue the Truly Blessed™ way is to ditch `Sync` and ask for a `LoggerFactory` instead.
